src/components/cdPipeline/NewCDPipeline.tsx

```diff
@@ -1074,6 +1074,7 @@ export default function NewCDPipeline({
                                     isWebhookCD={isWebhookCD}
                                     dockerRegistries={dockerRegistries}
                                     envIds={envIds}
+                                    noGitOpsModuleInstalledAndConfigured={noGitOpsModuleInstalledAndConfigured}
```

Comment:
Where is `noGitOpsModuleInstalledAndConfigured` defined?

Reply:
Fixed in Mukul's PR